stacksCollectible/stacksCollectionCache.ts

```diff
@@ -0,0 +1,30 @@
+import { StacksCollectionData } from 'index';
+
+type RecordType = { userNftCollectionData: StacksCollectionData[]; ttl: number };
+//global value stored to be used for in-memory cache
+export const nftCollection: Record<string, RecordType> = {};
+
+export function setCacheValue(key: string, userNftCollectionData: StacksCollectionData[]) {
+  const DEFAULT_TTL = 1 * 60 * 60 * 1000; // ttl for 1 hour
```

Comment:
I think 1 hour might be too long for mobile. consider the following use cases:

user is navigating between tabs, and paginating stx NFTs
user has just bought a new NFT, and wants to see it

1 hour TTL will cover 1. but not be great for 2. on mobile. user would have to close the app and restart it to force a refetch

5mins TTL should still cover 1. but also 2.

Reply:
ok makes sense. then we should have a separate TTL for mobile (5min) and extension (1hr)  right ?

Reply:
there's no real downside of using 5mins TTL for both. extension is very easily closed and reopened, so it would be an edge case for the cache to last over 5mins anyway